### PR TITLE
[MM-28080] scripts/test: do not remove existing data directory

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
This PR tweaks the `test.sh` file to not to remove `./data` directory before running the tests.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28080

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ PACKAGES_COMMA=$(echo $PACKAGES | tr ' ' ',')
 
 echo "Packages to test: $PACKAGES"
 find . -name 'cprofile*.out' -exec sh -c 'rm "{}"' \;
-find . -type d -name data -not -path './vendor/*' | xargs rm -rf
+find . -type d -name data -not -path './vendor/*' -not -path './data'| xargs rm -rf
```
Comment (lieut-data) [inline scripts/test.sh:17]: nit:
```suggestion
 find . -type d -name data -not -path './vendor/*' -not -path './data' | xargs rm -rf
```

Comment (jespino) [reply scripts/test.sh]: Why the extra space at the beginning?

Comment (isacikgoz) [reply scripts/test.sh]: Hmm, I overlooked it, probably a typo.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**